frontend/javascripts/oxalis/view/layouting/default_layout_configs.js

```diff
@@ -127,13 +127,15 @@ const globalLayoutSettings: GlobalConfig = {
   splitterSize: defaultSplitterSize,
   tabEnableRename: false,
   tabEnableClose: false,
-  tabSetHeaderHeight: 20,
-  tabSetTabStripHeight: 20,
+  tabSetHeaderHeight: layoutHeaderHeight,
+  tabSetTabStripHeight: layoutHeaderHeight,
 };
 
 const subLayoutGlobalSettings: GlobalConfig = {
   ...globalLayoutSettings,
   tabSetEnableDivide: false,
+  tabSetHeaderHeight: layoutHeaderHeight + 6,
```

Comment:
Why + 6? :see_no_evil: Maybe add a comment which explains the magic here.

Reply:
I created the constant `additionalHeaderHeightForBorderTabSets` for the `6`. I hope this is explains the `+ 6 ` good enough :)